metadata_backend/database/db_service.py

```diff
@@ -142,6 +147,7 @@ def query(self, collection: str, query: Dict) -> AsyncIOMotorCursor:
         :param query: query to be used
         :returns: Async cursor instance which should be awaited when iterating
         """
+        LOG.debug("DB doc query performed.")
```

Comment:
Query doesn't actually happen here, it is done when data is returned from function here: https://github.com/CSCfi/metadata-submitter/blob/ea72ccddf7c9cfbd1aa2833adf773a7e6e8ba8d8/metadata_backend/api/operators.py#L213

Maybe move logging there?

Reply:
it was meant to signal that a query was called, but will add logging there as suggested.

Reply:
a bit confused on what to add there because there is already logs ... :confused:

Reply:
you're right, my bad. This is ok now